x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/anomalies/anomalies.tsx

```diff
@@ -20,16 +21,15 @@ export const Anomalies = () => {
   const { isActiveTab } = useTabSwitcherContext();
   const { request$ } = useRequestObservable();
   const { getParsedDateRange } = useDatePickerContext();
-  const { asset, overrides } = useAssetDetailsRenderPropsContext();
+  const { asset, renderMode } = useAssetDetailsRenderPropsContext();
   const { kibanaVersion, isCloudEnv, isServerlessEnv } = useContext(KibanaEnvironmentContext);
-  const { onClose = () => {} } = overrides?.anomalies ?? {};
 
   const parsedDateRange = useMemo(() => getParsedDateRange(), [getParsedDateRange]);
 
   return (
     <div ref={ref}>
       <AnomaliesTable
-        closeFlyout={onClose}
+        closeFlyout={(renderMode as FlyoutProps).closeFlyout}
```

Comment:
`closeFlyout` function is passed as props because there are two different fly-outs where `Anomalies-table` is displayed (first is host fly-out view and second is ML anomaly detection fly-out).

I agree about `renderMode.mode` check. Updated.

Reply:
This is not applicable anymore. `closeFlyout` removed from `anomalies.tsx`